src/crate/theme/rtd/crate/static/vendor/furo/styles/variables/_index.sass

```diff
@@ -5,4 +5,4 @@
 @import "spacing"
 @import "icons"
 @import "admonitions"
-//@import "colors"
+@import "colors"
```

Comment:
Wow, excellent! 💯 Kudos for converging the code base to be even closer to Furo, now also importing its SASS- or SCSS-based color definitions.